### DISCUSSION TITLE
SigNoz

### DISCUSSION THREAD

**Discussion by GauthierPLM** [coollabsio/coolify#4132] [Service Template Requests]
SigNoz is a modern and open source observability platform leveraging the Open Telemetry protocol to integrate a wide range of languages and frameworks.

SigNoz requires to expose many ports & setup various volumes, so having a template for it would be very helpful.

Relevant issue on their Github: https://github.com/SigNoz/signoz/issues/6368

**Comment by galaczi** [comment]
This would be a great addition!

**Comment by GauthierPLM** [comment]
I opened a PR with a first version of a Signoz template: https://github.com/coollabsio/coolify/pull/5386

As some parts are not fully working yet, any help is welcome. In particular, I am not sure my approach is the best one to expose the OTEL collector.

**Comment by mredodos** [comment]
+1

**Comment by galaczi** [comment]
Looks like it's already added: https://coolify.io/docs/services/signoz
